### ISSUE TITLE
Allow overriding severity based on error code prefix match

### ISSUE THREAD

**Issue by jnooree** [microsoft/vscode-flake8#195]
Pycodestyle error codes (which are reported by flake8) are grouped by prefix: `E1xx` for indentation, `E2xx` for whitespaces, etc.
Currently, this extension could override flake8 severity only if full error code matches. It would be great to specify such "prefix" in config, and override all matching error codes at once. This also matches the behavior of flake8 itself.

For example, the following config:

```json
{
    "flake8.severity": {
        "E": "Warning",
        "E1": "Information",
    }
}
```

Will report all `Exxx` errors as warnings, and all indentation related errors (`E1xx`) as information.

**Comment by karthiknadig** [comment]
Duplicate of https://github.com/microsoft/vscode-flake8/issues/7

See https://github.com/microsoft/vscode-flake8/issues/7 for more details on this.

**Comment by karthiknadig** [comment]
@jnooree The code for this can be found here:
https://github.com/microsoft/vscode-flake8/blob/6f638010c80d4ee17a4ad515c3666f6d7cea7796/bundled/tool/lsp_server.py#L125-L135

We are open to accept PRs on this :)
